src/components/entity/state-badge.ts

```diff
@@ -15,6 +15,7 @@ import { computeActiveState } from "../../common/entity/compute_active_state";
 import { computeStateDomain } from "../../common/entity/compute_state_domain";
 import { stateIcon } from "../../common/entity/state_icon";
 import { iconColorCSS } from "../../common/style/icon_color_css";
+import { getRgbColor, LightEntity } from "../../data/light";
```

Comment:
Rename `getRgbColor` to identicate it's operating on a light entity. When we use auto complete, this might otherwise be lost.